QuizApp/Presentation/Home/HomeViewModel.swift

```diff
@@ -80,6 +97,19 @@ class HomeViewModel {
         isErrorPlaceholderVisible = true
     }
 
+    func switchFiltering(for mode: QuizzesFilteringMode) {
+        switch mode {
+        case .home:
+            guard let lastSelectedCategory = lastSelectedCategory else { return filteredQuizes = quizes }
```

Comment:
Why `return filteredQuizes = quizes`?

Reply:
when there is no selected category, reset filtered quizzes to all quizzes. Maybe return should have been moved after that statement :/